<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compte_medecin</class>
 <widget class="QMainWindow" name="compte_medecin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des medecins</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>ordonnance</string>
     </attribute>
     <widget class="QTabWidget" name="rechercher_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter une ordonnance</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>60</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordonnance</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>410</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>410</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicamen1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicament2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>295</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicamnet3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nomdocteur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>350</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>medicament4</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="doc_2">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom_2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med1_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med2_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>250</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med3_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>310</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="med4_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>350</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="picture_4">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>5</y>
          <width>931</width>
          <height>501</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>nump</string>
        </property>
       </widget>
       <zorder>picture_4</zorder>
       <zorder>label_12</zorder>
       <zorder>label_13</zorder>
       <zorder>pb_ajouter_2</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>label_14</zorder>
       <zorder>label_15</zorder>
       <zorder>label_16</zorder>
       <zorder>label_17</zorder>
       <zorder>label_18</zorder>
       <zorder>label_19</zorder>
       <zorder>num_3</zorder>
       <zorder>doc_2</zorder>
       <zorder>nom_2</zorder>
       <zorder>prenom_2</zorder>
       <zorder>med1_2</zorder>
       <zorder>med2_2</zorder>
       <zorder>med3_2</zorder>
       <zorder>med4_2</zorder>
       <zorder>lineEdit_2</zorder>
       <zorder>label_20</zorder>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher les ordonnance</string>
       </attribute>
       <widget class="QTableView" name="tabordo_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>931</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="numm_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordonnance</string>
        </property>
       </widget>
       <widget class="QLabel" name="picture_5">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>5</y>
          <width>941</width>
          <height>491</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>440</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
       <zorder>picture_5</zorder>
       <zorder>tabordo_2</zorder>
       <zorder>numm_2</zorder>
       <zorder>label_21</zorder>
       <zorder>pdf_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Supprimer une ordonnance</string>
       </attribute>
       <widget class="QLineEdit" name="num_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>160</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>numordo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>240</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="picture_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>5</y>
          <width>941</width>
          <height>491</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>picture_6</zorder>
       <zorder>num_4</zorder>
       <zorder>label_22</zorder>
       <zorder>supprimer_2</zorder>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
